python/__init__.py

```diff
@@ -72,6 +72,27 @@ def __hash__(self):
 except ImportError:
     pass
 
+# Define a `with edit(layer)` statement
```

Comment:
Yeah but there is a `__init__` in the core folder so I think it might be better in there.